From a3e2bb4eba7530909dc6dff13872820d4ce0eed7 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 3 Jun 2015 13:18:51 +0200
Subject: [PATCH] dracut-functions: degrade message about optional missing
 install

degrade the message about missing optional install items from a warning
to an info.
---
 dracut-functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions b/dracut-functions
index 97f55f6..2aa1c6a 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -441,7 +441,7 @@ dracut_install() {
     while (($# > 0)); do
 	if ! inst "$1" ; then
 	    if [[ $optional = yes ]]; then
-		dwarning "Skipping program $1 as it cannot be found and is flagged to be optional"
+		dinfo "Skipping program $1 as it cannot be found and is flagged to be optional"
 	    else
 		derror "Failed to install $1"
 		exit 1
